falcon/request.py

```diff
@@ -1419,8 +1425,8 @@ def get_param_as_bool(self, name, required=False, store=None,
                 val = True
             elif val in FALSE_STRINGS:
```

Comment:
Unrelated side note: I checked that we could improve perf of the whole function by ~5% or so by using ``frozenset`` versions of ``TRUE_STRINGS`` and ``FALSE_STRINGS``, but we can that in another PR.

Reply:
Great suggestion!